Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -8,6 +8,23 @@
 
 import Foundation
 
+// MARK: - Properties
+public extension URL {
+    
+    /// SwifterSwift: Dictionary of the URL's query parameters
+    public var queryParameters: [String: String?]? {
```

Comment:
Can't the key just be of type `String`? Looking up a value in a dictionary returns an optional value by default